About.html

```diff
@@ -28,41 +28,49 @@
   </header>
 
   <header class="navDesktop">
-    <nav class="navbar bg-dark">
-      <div class="container-fluid">
+    <div>
+      <ul class="top-nav desktop p-2">
+          <li class="socialMedia"><a class="icon" href="https://github.com/HectorTorresE"><i class="bi bi-github text-white"></i></a></li>
+          <li class="socialMedia"><a class="icon" href="https://www.linkedin.com"><i class="bi bi-linkedin text-white"></i></a></li>
+          <li class="socialMedia"><a class="icon" href="https://twitter.com/HectorT00406915"><i class="bi bi-twitter text-white"></i></a></li>
+          <li><a class="link">English</a></li>
+          <li><a class="link">My Page</a></li>
+          <li><a class="link">Logout</a></li>
+      </ul>
+  </div>
+    <nav class="nav-bar">
         <div class="row">
-          <div class="col-4">
-            <a class="title" href="#">TDM Festival</a>
+          <div class="col-4 text-left">
```

Comment:
## [OPTIONAL]
- [ ] Great job so far +1 but I think you can make your logo to be a link direct to the home page so that every time a user will make a click, the home page will show up.